vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListFragment.kt

```diff
@@ -166,7 +168,7 @@ class HomeRoomListFragment @Inject constructor(
         showFABs()
 
         views.newLayoutCreateChatButton.setOnClickListener {
-            // Click action for create chat modal goes here (Issue #6717)
+            newChatBottomSheet.show(requireActivity().supportFragmentManager, NewChatBottomSheet.TAG)
```

Comment:
Nit: Wdyt about creating the bottom sheet instance here instead of a class variable?